Extension/src/packageVersion.ts

```diff
@@ -14,7 +18,7 @@ export class PackageVersion {
     constructor(version: string) {
         const tokens: string[] = version.split(new RegExp('[-\\.]', 'g')); // Match against dots and dashes
         if (tokens.length < 3) {
-            throw new Error('Failed to parse version string: ' + version);
+            throw new Error(localize("version.string.parse.failed", "Failed to parse version string: {0}", version));
```

Comment:
I believe that these errors are only for telemetry (in connection with `githubapi.ts` for the Insiders updateChannel)